tools/sdk-testgen/.npmrc

```diff
@@ -0,0 +1,2 @@
+registry=https://registry.npmjs.org/
```

Comment:
Why is this needed? I wouldn't expect it to be necessary.

Reply:
Removed it.